### PR TITLE
Tempo: Trace to metrics docs

### PR DESCRIPTION
**Pull request by joey-grafana**
**What is this feature?**

Adds more verbose trace to metrics docs.

**Why do we need this feature?**

So users know more about what they can do with this feature.

**Who is this feature for?**

Tempo users.

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/datasources/tempo/configure-tempo-data-source.md [hunk 1]
```diff
@@ -69,27 +69,11 @@ There are two ways to configure the trace to logs feature:
...
+1. Optionally select tags to map. These tags can be used in the custom query with `${__tags}` variable. This variable will interpolate the mapped tags as list in an appropriate syntax for the data source and will only include the tags that were present in the span omitting those that weren't present. You can optionally configure a new name for the tag. This is useful in cases where the tag has dots in the name and the target data source does not allow using dots in labels. For example, you can remap `http.status` to `http_status` in such a case. If you don't map any tags here, you can still use any tag in the query like this `method="${__span.tags.method}"`. You can learn more about custom query variables [here](/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#custom-query-variables).
```
Comment (knylander-grafana) [inline docs/sources/datasources/tempo/configure-tempo-data-source.md:72]:
```suggestion
1. Optional: Select tags to map. These tags can be used in the custom query with `${__tags}` variable. This variable interpolates the mapped tags as list in an appropriate syntax for the data source. Only the tags that were present in the span are included; tags that aren't present are omitted You can also configure a new name for the tag. This is useful in cases where the tag has dots in the name and the target data source doesn't allow dots in labels. For example, you can remap `http.status` to `http_status`. If you don't map any tags here, you can still use any tag in the query, for example, `method="${__span.tags.method}"`. You can learn more about [custom query variables](/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#custom-query-variables).
```

File: docs/sources/datasources/tempo/configure-tempo-data-source.md [hunk 2]
```diff
@@ -116,32 +100,77 @@ The **Trace to metrics** setting configures the [trace to metrics feature](/blog
...
-- **Query:** The query ran when navigating from a trace to the metrics data source.
-  Interpolate tags using the `$__tags` keyword.
-  For example, when you configure the query `requests_total{$__tags}`with the tags `k8s.pod=pod` and `cluster`, the result looks like `requests_total{pod="nginx-554b9", cluster="us-east-1"}`.
+   These tags can be used in the custom query with `${__tags}` variable. This variable interpolates the mapped tags as list in an appropriate syntax for the data source and will only include the tags that were present in the span omitting those that weren’t present. You can optionally configure a new name for the tag. This is useful in cases where the tag has dots in the name and the target data source doesn't allow using dots in labels. For example, you can remap `service.name` to `service_name` in such a case. If you don’t map any tags here, you can still use any tag in the query like this `method="${__span.tags.method}"`. You can learn more about custom query variables [here](/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#custom-query-variables).
+
+1. Click the **Add query** button to add a custom query.
```
Comment (knylander-grafana) [inline docs/sources/datasources/tempo/configure-tempo-data-source.md:129]: Bold indicates that this is a UI element, so we don't need the  and button. 
```suggestion
1. Click **Add query** to add a custom query.
```

File: docs/sources/shared/datasources/tempo-traces-to-profiles.md [hunk 3]
```diff
@@ -66,29 +66,13 @@ To use a custom query with the configuration, follow these steps:
...
+   These tags can be used in the custom query with `${__tags}` variable. This variable interpolates the mapped tags as list in an appropriate syntax for the data source. Only the tags that were present in the span are included; tags that aren't present are omitted. You can also configure a new name for the tag. This is useful in cases where the tag has dots in the name and the target data source doesn't allow using dots in labels. For example, you can remap `service.name` to `service_name`. If you don’t map any tags here, you can still use any tag in the query, for example: `method="${__span.tags.method}"`. You can learn more about custom query variables [here](/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#custom-query-variables).
```
Comment (knylander-grafana) [inline docs/sources/shared/datasources/tempo-traces-to-profiles.md:69]:
```suggestion
   These tags can be used in the custom query with `${__tags}` variable. This variable interpolates the mapped tags as list in an appropriate syntax for the data source. Only the tags that were present in the span are included; tags that aren't present are omitted. You can also configure a new name for the tag. This is useful in cases where the tag has dots in the name and the target data source doesn't allow using dots in labels. For example, you can remap `service.name` to `service_name`. If you don’t map any tags here, you can still use any tag in the query, for example: `method="${__span.tags.method}"`. You can learn more about [custom query variables](/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#custom-query-variables).
```

**Review by knylander-grafana — APPROVED** [review]
Thank you for updating the doc! I've made a few suggested changes.

**Review by aocenas — APPROVED** [review]

**Pull request merged**